pkg/sqlcmd/commands.go

```diff
@@ -282,6 +288,24 @@ func errorCommand(s *Sqlcmd, args []string, line uint) error {
 	return nil
 }
 
+func onerrorCommand(s *Sqlcmd, args []string, line uint) error {
+	if len(args) == 0 || args[0] == "" {
+		return InvalidCommandError("ON ERROR", line)
+	}
+	params := strings.TrimSpace(args[0])
+	if params == "" {
+		s.Connect.ExitOnError = true

```

Comment:
Shouldn't we raise an error in this case if no 'exit' or 'ignore' is specified here?